src/cpp/README.md

```diff
@@ -9,6 +9,23 @@ This section describes how to add gRPC as a dependency to your C++ project.
 In the C++ world, there's no universally accepted standard for managing project dependencies.
 Therefore, gRPC supports several major build systems, which should satisfy most users.
 
+## Supported Platforms
+
+* Supported: These platforms are officially supported. We test our code on these
+platform and have automated continuous integration tests for them.
+
+* Best Effort: We do not have continous integration tests for these, but we are fairly confident that gRPC C++ would work on them.
+We will make our best effort to support them, and we welcome patches for such platforms, but we might need to declare bankruptcy on some issues.
+
+| Operating System | Architectures | Versions | Support Level |
+|------------------|---------------|----------|---------------|
+| Linux            | x86, x64      | clang 3.4+, GCC 4.9+ | Supported |
+| Windows 7+       | x86, x64      | Visual Studio 2015+  | Supported |
```

Comment:
Windows 7 is actually end of life since January 2020, so we might as well drop its official support from the list. Windows 10+ is enough.